### DISCUSSION TITLE
Ventura - Stitch Fix  Mac os

### DISCUSSION THREAD

**Discussion by Stitchaguliz** [inkstitch/inkstitch#2079] [General]
is there a version of INK Stitch for Ventura 13.2.1?

**Comment by stitcher99** [comment]
I cant seem to install ink stitch for my mac which is updated to ventura. :/

**Comment by kaalleen** [comment]
Ventura isn't recommended for Inkscape because certain mouse events are unresponsive. But the installation of Inkscape and then Ink/Stitch should be working.
Can you describe what you have tried so far and if you see any error messages during installation?
